flocker/acceptance/testtools.py

```diff
@@ -928,6 +928,35 @@ def created(response):
     return creating
 
 
+def extract_external_port(
+    client, container_identifier, internal_port
+):
+    """
+    Inspect a running container for the external port number on which a
+    particular internal port is exposed.
+
+    :param docker.Client client: The Docker client to use to perform the
+        inspect.
```

Comment:
Why is this in terms of the docker client, rather than our own interface around the docker client?